public/app/plugins/datasource/prometheus/result_transformer.ts

```diff
@@ -57,7 +64,23 @@ export function transform(
   };
   const prometheusResult = response.data.data;
 
-  if (!prometheusResult.result) {
+  if (isExemplarData(prometheusResult)) {
+    const events: AnnotationEvent[] = [];
+    prometheusResult.forEach(exemplarData => {
+      const data = exemplarData.exemplars.map(exemplar => {
+        return {
+          time: exemplar.timestamp,
+          text: getText(exemplar, transformOptions.exemplarTraceIDDestination),
```

Comment:
do exemplars have a Y value?  It is actually helpful to use the `AnnotationEvent` interface here?

Reply:
> do exemplars have a Y value?
They have a timestamp and a value so I assume that's what you mean by y value.
> It is actually helpful to use the AnnotationEvent interface here?
What do you mean? It helps with intellisense and type checking for sure.,